src/test/common/services/createUserApi.test.js

```diff
@@ -5,16 +5,92 @@ const { expect } = require('chai');
 const chai = require('chai');
 const sinonChai = require('sinon-chai');
 const proxyquire = require('proxyquire');
-const { URL } = require('url');
+const nock = require('nock');
 
 require('../../global.test');
+const endpoints = require('../../../common/config/endpoints');
+const registerApi = require('../../../common/services/createUserApi');
```

Comment:
This originally was in users.test.js, and whilst true in a sense, it makes more sense to group tests by the class they are in, so the createUserApi based nock tests are now here.